client-go/cmd/certs.go

```diff
@@ -0,0 +1,143 @@
+package cmd
+
+import (
+	"fmt"
+	"io/ioutil"
+	"strings"
+
+	"github.com/deis/deis/pkg/prettyprint"
+
+	"github.com/deis/deis/client-go/controller/client"
+	"github.com/deis/deis/client-go/controller/models/certs"
+)
+
+// CertsList lists certs registered with the controller.
+func CertsList() error {
+	c, err := client.New()
+
+	if err != nil {
+		return err
+	}
+
+	certList, err := certs.List(c)
+
+	if err != nil {
+		return err
+	}
+
+	if len(certList) == 0 {
+		fmt.Println("No certs")
+		return nil
+	}
+
+	certMap := make(map[string]string)
+	nameMax := 0
+	expiresMax := 0
+	for _, cert := range certList {
+		certMap[cert.Name] = cert.Expires
+
+		if len(cert.Name) > nameMax {
+			nameMax = len(cert.Name)
+		}
+		if len(cert.Expires) > nameMax {
+			expiresMax = len(cert.Expires)
+		}
+	}
+
+	nameHeader := "Common Name"
+	expiresHeader := "Expires"
+	tabSpaces := 5
+	bufferSpaces := tabSpaces
+
+	if nameMax < len(nameHeader) {
+		tabSpaces += len(nameHeader) - nameMax
+		nameMax = len(nameHeader)
+	} else {
+		bufferSpaces += nameMax - len(nameHeader)
+	}
+
+	if expiresMax < len(expiresHeader) {
+		expiresMax = len(expiresHeader)
+	}
+
+	fmt.Printf("%s%s%s\n", nameHeader, strings.Repeat(" ", bufferSpaces), expiresHeader)
+	fmt.Printf("%s%s%s\n", strings.Repeat("-", nameMax), strings.Repeat(" ", 5),
+		strings.Repeat("-", expiresMax))
+	fmt.Print(prettyprint.PrettyTabs(certMap, tabSpaces))
+	return nil
+}
+
+// CertAdd adds a certs to the controller.
+func CertAdd(cert, key, commonName, sans string) error {
+	c, err := client.New()
+
+	if err != nil {
+		return err
+	}
+
+	fmt.Print("Adding SSL endpoint... ")
+	quit := progress()
+	err = processCertsAdd(c, cert, key, commonName, sans)
+	quit <- true
+	<-quit
+
+	if err != nil {
+		return err
+	}
+
+	fmt.Println("done")
+	return nil
+}
+
+func processCertsAdd(c *client.Client, cert string, key string, cN string, sans string) error {
```

Comment:
I would shorten the func signature this way, although this is just personal taste:
```go
func processCertsAdd(c *client.Client, cert, key, cN, sans string) error
```